README.rst

```diff
@@ -10,7 +10,7 @@ Description
 
 This repository contains a Python script (and executable) that allows one to download MODIS data granules for different products and periods. 
 
-The code is quite simple and generic, and should work with most standard Python installations.
+The code is quite simple and generic but requires at least a couple of specific libraries not usually available by default: Beautiful Soup (conda-forge: `bs4`) and lxml (conda-forge: `lxml`).
```

Comment:
I'm usually wary of adding dependencies for simple tasks such as scanning a few links. I see it simplifies some of the code somewhat, but... I'm already wary of using `requests` as I couldn't get urllib to work with redirection.

Reply:
The returned html is relatively complicated, especially for doing the earthdata login, so this seems a reasonable compromise to me.